### PR TITLE
Add resume info input section

### PR DESCRIPTION
**Pull request by dlwlstks96**
feat : Add resume info input section( #5 )

image

We can get name, birth, phone number, address, and work experience.
And the design will be revised after we decide on the concept.
If you have any corrections or additions, please give me feedback.
@6loss0m @lin9703 @dd-create

### CODE REVIEW

**Comment by 6loss0m** [comment]
I think it's better to add a few for ease of entry.
+ Adding a calendar to the date of birth
+ Add Selections to Email
     ex) sss@naver.com / sss@gmail.com

File: app/src/main/java/com/euphonyforever/looking_for_job/InputResumeInfo.java [hunk 1]
```diff
@@ -32,10 +37,11 @@ protected void onCreate(Bundle savedInstanceState) {
             @Override
             public void onClick(View view) {
                 String name = editTextName.getText().toString();
-                String birth = editTextBirth.getText().toString();
-                String number = editTextNumber.getText().toString();
-                String address = editTextAddress.getText().toString();
-                String career = editTextCareer.getText().toString();
+                name = editTextName.getText().toString();
```
Comment (lin9703) [inline app/src/main/java/com/euphonyforever/looking_for_job/InputResumeInfo.java:40]: Duplicate name.

**Comment by dlwlstks96** [comment]
The access authority has been modified.
The part below will be further corrected later.

> I think it's better to add a few for ease of entry.
> 
> * Adding a calendar to the date of birth
> * Add Selections to Email
>   ex) [sss@naver.com](mailto:sss@naver.com) / [sss@gmail.com](mailto:sss@gmail.com)

File: app/src/main/java/com/euphonyforever/looking_for_job/InputResumeInfo.java (hunk 1 above)
Comment (dlwlstks96) [reply app/src/main/java/com/euphonyforever/looking_for_job/InputResumeInfo.java:39]: Edit done!

**Pull request merged**